apps/datahub/src/app/dataset/dataset-information/dataset-information.component.ts

```diff
@@ -31,4 +41,10 @@ export class DatasetInformationComponent {
 
     return categoryKeywords?.map((keyword) => keyword.label)
   }
+
+  onInfoKeywordClick(keyword: string) {
+    this.routerFacade
+      ? this.routerFacade.updateSearch({ q: keyword })
+      : goFromHomeToSearch(keyword)
```

Comment:
This function is only needed to navigate from the `accueil` app to the `catalogue` app, so it can be removed here along with the `routerFacade`.